src/app/(proper_react)/DashboardTopBanner.module.scss

```
@@ -0,0 +1,41 @@
+@import "../tokens";
+
+.container {
+  display: flex;
+  flex-direction: row;
+  position: relative;
+  border: 1px solid rgba($color-purple-50, 0.2);
+  border-radius: $border-radius-md;
+  padding: $layout-lg $spacing-xl;
+
+  .content {
+    display: flex;
+    flex-direction: column;
+    gap: $spacing-sm;
+
+    h3 {
+      font: $text-title-2xs;
+    }
+
+    .cta {
+      margin-top: $spacing-sm;
+      align-self: start;
+    }
+  }
+
+  & > * {
+    flex: 1;
```

I'd either set all properties with the short-hand, or only the property you're intending to set, to make it easier to understand for readers. That said, I'm not sure what effect this line has if it's applied to every child?

```suggestion
    flex-grow: 1;
```

---

Right, so then the values that are implicitly set by leaving them out (specifically: `flex-shrink: 1`) are relevant as well - so I'd still set those explicitly, so it's clear to the reader that you're doing that. So I'd suggest either

```suggestion
    flex: 1 1 auto;
```

 (assuming `flex-basis` doesn't actually have to be 0)

or

```suggestion
    flex-grow: 1;
    flex-shrink: 1;
```